code/modules/projectiles/guns/ballistic/pistol.dm

```diff
@@ -9,6 +9,10 @@
 	fire_delay = 0
 	actions_types = list()
 
+/obj/item/gun/ballistic/automatic/pistol/shoot_live_shot(mob/living/user as mob|obj, pointblank = 0, mob/pbtarget = null, message = 1)
+	user.changeNext_move(3.5) //deciseconds delay after click, default is 4
+	. = ..()
```

Comment:
The server and byond run on (by default) decisecond-long ticks, and mostly ignores values betwixt. That value could be easily floored up by the game, thus differing in no way from the standard 4 seconds cooldown if the modifiers are left untouched.
Even if that was not the case, https://github.com/Citadel-Station-13/Citadel-Station-13/blob/5d2678a2b615ef2a59c7c0a06c8343fb62c76caa/code/modules/projectiles/ammunition/_firing.dm#L18 will override the click cooldown anyway, making this change useless.